### ISSUE TITLE
Rename BF10 when comparing to best model?

### ISSUE THREAD

**Issue by quentingronau** [jasp-stats/jasp-issues#129]
Personally, I find it a bit confusing that when the option "Compare to best model" is selected, the output table says "BF10" which is in fact not the Bayes factor against the null model but against the best model. Maybe we could change this in the tables/interface to something more informative?

**Comment by juliuspfadt** [comment]
@EJWagenmakers, any thoughts?

**Comment by JohnnyDoorn** [comment]
This is indeed a confusing thing!
My suggestion: replace 1 and 0 with R and B (or 0, if comparing to null) and add a footnote to explain the subscript:
![image](https://user-images.githubusercontent.com/15704203/232002927-3be7b4ce-4d24-4de7-9da4-ddd035806b89.png)
So here we would have BF_{R, B} as the column header, and add a footnote along the lines of 

> BF{R, B} denotes the model comparison between the model in each row (R), and the best model (B). 

and if comparing to null model

> BF{R, 0} denotes the model comparison between the model in each row (R), and the null model (0).


@vandenman do you agree? Then I can update the code (also for Bayesian regression?).

**Comment by vandenman** [comment]
I agree with the change, and in fact wanted to implement this sometime in the past. It's a bit more difficult to implement than you'd think though. The main issue is that we use Unicode for subscript 0 and 1 but unfortunately the subscript b does not exist in Unicode... (see https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts#Latin,_Greek_and_Cyrillic_tables). Instead you have to somehow use html tags but those are (intentionally) removed in jaspTables so this will require some more work. 

@JorisGoosen do you remember what needs to be done to allow for HTML tags in jaspTables?

**Comment by JorisGoosen** [comment]
We could simply add a workaround for the HTML removal part to not remove those subscripts?

**Comment by JorisGoosen** [comment]
Only thing necessary is changing `escapeHtmlStuff` to:
```
	inline static std::string escapeHtmlStuff(std::string input)
	{
		input = replaceBy(input, "&", "&amp;");
		input = replaceBy(input, "<", "&lt;");
		input = replaceBy(input, ">", "&gt;");
		input = replaceBy(input, "&lt;sub&gt");
		input = replaceBy(input, "&lt;sub/&gt");

		return input;
	}
```

**Comment by Kucharssim** [comment]
Another alternative would be to implement LaTeX expressions in our output. That would be nice and enable much more cool functionality, but since my light testing of KaTeX (https://github.com/jasp-stats/INTERNAL-jasp/issues/395) I did not have any time to push this forward.

**Comment by vandenman** [comment]
> Another alternative would be to implement LaTeX expressions in our output.

That would be much more complicated, although it would allow for some nicer output. I think for the short-term we could create a function like `asHTML("text")` which sets an attribute on the string that tells jaspTable it should not run `escapeHtmlStuff` on it. That way we can also reuse it when setting data, e.g., 
```r
tb <- createJaspTable()
tb$addColumnInfo(name = "data", title = asHTML("BF<sub><B1/sub>"))
tb[["data"]] = asHTML(paste0("data<sub>", letters, "</sub>"))
```
which would then show up like so
|  BF<sub>B1</sub>  |
|-------------------|
| data<sub>a</sub>  |
| data<sub>b</sub>  |
|       ...         |
| data<sub>y</sub>  |
| data<sub>z</sub>  |

alternatively, we could add two arguments to `$addColumnInfo` like `escapeHTMLinTitle` and `escapeHTMLinData`. 

@Kucharssim any preferences for either approach? perhaps adding two more arguments to `$addColumnInfo` is more discoverable?

**Comment by Kucharssim** [comment]
> That would be much more complicated

Well it doesn't have to be that difficult, see https://github.com/jasp-stats/jasp-desktop/pull/5073. Considering @shun2wang picked this up and it looks like we could potentially have LaTeX support already for the next release, I would prefer to handle it that way than adding short-term solutions, otherwise we'll end up having to support both approaches. But I don't mind too much if we go this way - it obviously depends on how pressing this issue is.

**Comment by Kucharssim** [comment]
> Well, I'm not sure if that will go into 0.17.2

definitely not in 0.17.2, I meant the one after that

**Comment by JorisGoosen** [comment]
The fix I mention in https://github.com/jasp-stats/jasp-issues/issues/129#issuecomment-1508261611 should be real easy, low errorchance and should allow for `<sub>` and `</sub>` in jaspHtml/jaspTable/etc without it getting messed up.

I can do this in like 5 minutes right now for 0.17.2. And I can also add `<sup>` and `</sup>`
Then for 0.17.3 or 0.18 we have katex support as well, but these fixes should only have as a sideeffect that people who have `<sup>` etc in their data might get some missing output?

**Comment by shun2wang** [comment]
In katex I seted only render a formula with` $$formula$$ `or `\\begin formula \\end` as elements to be render which will including into a span class name of `katex` `katex-mathml` `katex-html` things, so I am not sure if ` <sup>`  will be missing Maybe I misunderstood the meaning?

**Comment by JorisGoosen** [comment]
The `<sup>` thing is specifically for the quick fix I suggest in https://github.com/jasp-stats/jasp-issues/issues/129#issuecomment-1508261611 

Not related to katex

**Comment by JorisGoosen** [comment]
Ill just make a branch of this

**Comment by JorisGoosen** [comment]
<img width="170" alt="image" src="https://user-images.githubusercontent.com/29062713/234289842-45fcf2ec-98dc-4780-82cb-fff2d72eea7e.png">

**Comment by JorisGoosen** [comment]
<img width="1492" alt="image" src="https://user-images.githubusercontent.com/29062713/234305691-1baf720f-77a7-4914-ad26-7c0b7af3bac3.png">

**Comment by tomtomme** [comment]
@Kucharssim @JohnnyDoorn 
So it seems @JorisGoosen merged the patch needed to convert from BF10 to BFRB (see https://github.com/jasp-stats/jasp-issues/issues/129#issuecomment-1508213296)

But the tables in ANOVA and Regression still need to be updated to use that code and show BFRB.

**Comment by tomtomme** [comment]
still valid with current 0.19beta
